Sources/ZcashLightClientKit/Synchronizer/SDKSynchronizer.swift

```diff
@@ -394,9 +394,9 @@ public class SDKSynchronizer: Synchronizer {
         try throwIfUnprepared()
 
         // let's see if there are funds to shield
-        let accountIndex = Int(spendingKey.account)
+        let account = Zip32Account(Int32(spendingKey.account))
```

Comment:
```suggestion
        let accountIndex = spendingKey.accountIndex
```
(I am going to also suggest renaming `UnifiedSpendingKey.index` and changing its type to `Zip32AccountIndex` so that this works.)

Reply:
Updated